feat: implement different cable colors (#317)
refinedmods/refinedstorage2: CHANGELOG.md

## [Unreleased]

### Added

> - Support for different colors of cable, that only connect to same colored cable and other RS parts

Reviewer: ```suggestion
- Different Cable colors. They only connect to same colored cables or the default cable.
```
